dev-dotnet/dotnetcore-sdk-bin-common/dotnetcore-sdk-bin-common-3.1.100_pre1.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit eutils
+
+DESCRIPTION="Common files shared between multiple slots of .NET Core"
+HOMEPAGE="https://www.microsoft.com/net/core"
+LICENSE="MIT"
+
+SRC_URI="
+amd64? ( https://download.visualstudio.microsoft.com/download/pr/a3cc3d8a-226d-4306-a61b-a5446fdb72ef/604e029047aec0229545e8c397a14ddb/dotnet-sdk-3.1.100-preview1-014459-linux-x64.tar.gz )
+"
+
+SLOT="0"
+KEYWORDS=""
+
+QA_PREBUILT="*"
+RESTRICT="splitdebug"
+
+# The sdk includes the runtime-bin and aspnet-bin so prevent from installing at the same time
+# dotnetcore-sdk is the source based build
+
+RDEPEND="
+	=dev-dotnet/dotnetcore-sdk-bin-${PV}
```

Comment:
This is a circular dependency, isn't it?

Reply:
yes, but only at runtime. neither works without the other, and the common package works only if the same version (except for revisions) are installed of the main package, while the main package works if the common package is at least that version.